gui/packages/ubuntupro/test/pages/landscape/landscape_model_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:io';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:p4w_ms_store/p4w_ms_store_method_channel.dart';
+import 'package:ubuntupro/core/agent_api_client.dart';
+import 'package:ubuntupro/pages/landscape/landscape_model.dart';
+
+import 'landscape_model_test.mocks.dart';
+
+@GenerateMocks([AgentApiClient])
+void main() {
+  group('landscape config', () {
+    const pluginChannel = MethodChannelP4wMsStore.methodChannel;
+    final pluginMessenger =
+        TestWidgetsFlutterBinding.ensureInitialized().defaultBinaryMessenger;
+    // Resets the plugin message handler after each test.
+    tearDown(() {
+      pluginMessenger.setMockMethodCallHandler(pluginChannel, null);
+    });
```

Comment:
Leftovers after a copy-paste? You don't need that plugin in these tests, so this can just be removed.